blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupRunnable.java

```diff
@@ -107,10 +107,12 @@ public void run() {
             Granularity dstGran = srcGran.coarser();
             ColumnFamily<Locator, Long> dstCF = CassandraModel.getColumnFamily(rollupClass, dstGran);
 
-            //Run the validation for enums every 5 minutes, when data is being rolledup from full to 5m
-            if (rollupType == RollupType.ENUM && dstGran.equals(Granularity.MIN_5)) {
+            if (rollupType == RollupType.ENUM) {
                 singleRollupReadContext.getEnumMetricsMeter().mark();
-                enumValidatorExecutor.execute(new EnumValidator(Sets.newHashSet(rollupLocator)));
+                //Run the validation for enums every 5 minutes, when data is being rolled up from full to 5m
+                if (dstGran.equals(Granularity.MIN_5) && Configuration.getInstance().getBooleanProperty(CoreConfig.ENUM_VALIDATOR_ENABLED) == true) {
```

Comment:
There is no direct connection between validations and rollups.  We added to rollups out of convenience, but it increases the load on rollups.  If the rollup load gets too great, we may decide to move it elsewhere, and may want to turn it off temporarily here.